src/main/java/com/kolya/gym/config/BearerTokenFilter.java

```diff
@@ -0,0 +1,52 @@
+package com.kolya.gym.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.GenericFilterBean;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.Collections;
+
+@Component
+public class BearerTokenFilter extends GenericFilterBean {
+
+    @Value("${auth.bearer.token}")
+    private String bearerToken;
+
+    @Override
+    public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain) throws IOException, ServletException {
+        HttpServletRequest httpServletRequest = (HttpServletRequest) request;
+        HttpServletResponse httpResponse = (HttpServletResponse) response;
+        String authorizationHeader = httpServletRequest.getHeader("Authorization");
+
+        if (authorizationHeader == null || !authorizationHeader.startsWith("Bearer ")) {
+            sendError(httpResponse, HttpServletResponse.SC_FORBIDDEN, "Bearer token is missing");
+            return;
+        }
+
+        String token = authorizationHeader.substring(7);
```

Comment:
```suggestion
        String token = authorizationHeader.substring(TOKEN_IDENTIFICATION_BOUNDARY);
```
Or any other appropriate naming. 

Better not to use magic numbers. They tend to make code less readable.

Reply:
Fixed